network_wrangler/models/gtfs/gtfs.py

```diff
@@ -12,6 +14,55 @@
     StopTimesTable,
     TripsTable,
 )
+from .types import RouteType
```

Comment:
I tend to think that all of these route type categorizations should be configs since it isn't something inherant in the definition of GTFS itself and is an assumption that you are making.  I wont die on that hill though :-)